app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      include Wor::Paginate
+
+      # GET /api/v1/:user_id/rents
+      def index
+        render_paginated rents
+      end
+
+      # POST /api/v1/rents
+      def create
+        rent = Rent.create!(rent_params)
+        render json: rent
```

Comment:
It's a good  practice to return 201 as status code when the resource has been successfuly created

Reply:
done!